mlibrary/pride
# Removing `Pride.FieldTree.ValueBoolean` and moving `Pride.FieldTree.insideFieldNodes`.

# Overview
`Pride.FieldTree.ValueBoolean` was never called, so it has been removed. This means that `Pride.FieldTree.Field` was the only method using `Pride.FieldTree.insideFieldNodes`, which is just an array of strings. The array has been added directly to `Pride.FieldTree.Field`, and `Pride.FieldTree.insideFieldNodes` has been removed.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#pride-fieldtree-valueboolean-removal` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#pride-fieldtree-valueboolean-removal"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- _How to use the feature_.
- Make sure the PR is consistent in these browsers:
